cwd/editor.js

```diff
@@ -1,9 +1,14 @@
-const makeButton = (id, div, text) => {
+const makeButton = (id, div, text, ref) => {
```

Comment:
Yes, I am sorry we are not using React or something that would make this code so much less painful to write. :(

Reply:
I wonder if it could be worthwhile to use a React server for every page other than the main CWD page, but I guess that would require learning React entirely.

Reply:
How much would it take to rewrite things into React?

Reply:
I honestly feel like if we put our minds to it and I didn't have other distractions, we could write this whole page in a 2-hour office hours block.  (Or just any other 2-hour block we set aside) I luckily have old ode that handles setup and everything.  The hard part would be making one page served by a different server (React has its own server), but https://stackoverflow.com/a/37999813/2397924 looks like it's not too hard (praise Express!).